blog/2019-10-08-integrate-infinite-loading/index.md

```diff
@@ -59,7 +59,7 @@ query ($page: Int) {
 }
 ```
 
-This is a pretty typical way to do traditional pagintation, but we're not going to include a pager component in the UI.
+This is a pretty typical way to do traditional paginatation, but we're not going to include a pager component in the UI.
```

Comment:
Can you change `paginatation` to `pagination`? You also made a typo.

Reply:
Sorry about that! I changed it